schema_based_on_diagram.sql

```diff
@@ -0,0 +1,47 @@
+CREATE DATABASE clinic;
+
+CREATE TABLE patients (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    admitted_at timestamp,
+    patient_id BIGINT REFERENCES patients (id)
```

Comment:
- [ ] Kindly add the `status` column inside `medical_histories` table as per the schema diagram.

Reply:
Thank you for pointing that out. We will check it out.